cloud/aws/templates/aws_oidc/bin/aws_cli.py

```diff
@@ -53,9 +53,18 @@ def restart_ecs_service(self, cluster: str, service_name: str):
                 f'--service={service_name}', f'--cluster={cluster}'
             ])
 
+    def get_load_balancer_dns(self, name: str) -> str:
```

Comment:
we can get this as output variable from terraform https://registry.terraform.io/modules/cn-terraform/ecs-fargate-service/aws/latest?tab=outputs aws_lb_lb_dns_name

See https://www.terraform.io/cli/commands/output

Reply:
Yes we can. I don't see much benefit though. We already have to use AWS CLI for various stuff so for consistency I think it's better to use it. One benefit of using terraform is that we don't need to rely on LB name which is defined in terraform and hardcoded in python code. It doesn't seem like a that big of a problem though.